### ISSUE TITLE
Type inconsistency between Tensor.batchNormalized and BatchNorm

### ISSUE THREAD

**Issue by t-ae** [tensorflow/swift-apis#490]
`epsilon` of `Tensor.batchNormalized` is `Float`.
https://github.com/tensorflow/swift-apis/blob/b7f2a06a4c0225884a3ce2e7010d558d7038e7c5/Sources/TensorFlow/Operators/NN.swift#L36
But `epsilon` of `BatchNorm` is `Tensor<Float>`.
https://github.com/tensorflow/swift-apis/blob/b7f2a06a4c0225884a3ce2e7010d558d7038e7c5/Sources/TensorFlow/Layers/Normalization.swift#L34

`epsilon` is a scalar (even `BatchNorm`'s comment says so).
Changing `BatchNorm.epsilon` to `Float` improve API consistency.

My concern is that `BatchNorm` is aleready marked as `@frozen`. Is it OK to change this?

Also `LayerNorm` has `epsilon: Tensor<Float>` which is scalar indeed.

**Comment by sgugger** [comment]
This has been solved by #525 , so closing this issue.
